src/components/Checkbox.js

```diff
@@ -47,77 +47,77 @@ const defaultProps = {
     onMouseDown: undefined,
 };
 
-class Checkbox extends React.Component {
-    constructor(props) {
-        super(props);
+const Checkbox = (props) => {
```

Comment:
we have [standardized](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#should-i-declare-my-components-with-arrow-functions-const-or-the-function-keyword) using the `function` keyword for component declarations